docs/hooks/appointment-booking.md

```diff
@@ -0,0 +1,237 @@
+# `appointment-booking`
+
+| Metadata | Value
+| ---- | ----
+| specificationVersion | 1.0
+| hookVersion | 1.0
+
+## Workflow
+
+This hook is invoked when the user is scheduling one or more future encounters/visits for the patient.  It may be invoked at the start and end of the booking process and/or any time between those two points.  It allows hook services to intervene in the decision of when future appointments should be scheduled, where they should be scheduled, what services should be booked, to identify actions that need to occur prior to scheduled appointments, etc.
+
+## Context
+
+The Patient whose appointment is/appointments are bing booked as well as the draft Appointment records.
+
+Field | Optionality | Prefetch Token | Type | Description
+----- | -------- | ---- | ---- | ----
```

Comment:
Why not include `userId` in the context (like all of our other hooks) ?

Reply:
Added